server/apps/immich/src/app.module.ts

```diff
@@ -61,4 +61,9 @@ import { AuthGuard } from './middlewares/auth.guard';
   ],
   providers: [{ provide: APP_GUARD, useExisting: AuthGuard }, AuthGuard],
 })
-export class AppModule {}
+export class AppModule implements OnModuleInit {
+  constructor(private searchService: SearchService) {}
+  async onModuleInit() {
+    await this.searchService.bootstrap();
```

Comment:
We use this method because of the async/await calls, which cannot be done in a constructor, correct?

Reply:
The search service will be injected/created several place and we only need the bootstrap code to only run one in startup, so I put it here.